lib/main.dart

```diff
@@ -58,7 +61,9 @@ class ECellApp extends StatelessWidget {
               create: (_) => EventsCubit(FakeEventsRepository()), child: EventsScreen()),
           S.routeSponsors: (_) => BlocProvider(
               create: (_) => SponsorsCubit(APISponsorsRepository()), child: SponsorsScreen()),
-          S.routeEsummit: (_) => ESummitScreen()
+          S.routeEsummit: (_) => ESummitScreen(),
+          S.routeTeam: (_) =>
+              BlocProvider(create: (_) => TeamCubit(FakeTeamRepository()), child: TeamScreen()),
```

Comment:
TeamScreen is supposed to be a tab in about us section. so there should be a routeAboutUs routing us to AboutUsScreen().

Reply:
This is ondone.